.tav.yml

```diff
@@ -153,15 +138,13 @@ mongodb-core:
     mode: latest-minors
     include: '>=1.2.19 <4' # latest minors subset of '>=1.2.19 <4'
 
-mongodb-3:
-  name: mongodb
-  versions: '>=3.3 <4'
-  commands: node test/instrumentation/modules/mongodb/mongodb.test.js
-mongodb-4:
+mongodb-3-4:
   name: mongodb
-  versions: '>=4 <5'
-  node: '>=12.9.0'
+  versions: '3.3.0 || 3.3.5 || 3.4.1 || 3.5.11 || 3.6.12 || 3.7.4 || 4.0.1 || 4.1.4 || 4.2.2 || 4.3.1 || 4.4.1 || 4.5.0 || 4.6.0 || 4.7.0 || 4.8.1 || 4.9.1 || 4.10.0 || 4.11.0 || 4.12.1 || 4.13.0 || 4.14.0 || 4.15.0 || 4.16.0 || ^4.17.1'
   commands: node test/instrumentation/modules/mongodb/mongodb.test.js
+  update-versions:
+    mode: latest-minors
```

Comment:
nit: Similar to the above comment. Consider `mode: max-5` for this group.

Reply:
done!